docs/products/opensearch/howto/opensearch-search-and-python.rst

```diff
@@ -0,0 +1,558 @@
+    - Less than
+  * - ``lte``
+    - Less than or equal to
+
+
+Try to find recipes in a certain range of sodium, for example:
+
+::
+
+    python search.py range sodium 0 10
+
+.. seealso::
+
+  See more about the range query in the `OpenSearch documentation <https://opensearch.org/docs/latest/opensearch/query-dsl/term/#range>`_.
+
+.. _fuzzy-query:
+
+Fuzzy queries
+-------------
+This query looks for documents that have **similar term** to the searched term. This similarity is calculated by the ``Levenshtein`` `edit distance <https://en.wikipedia.org/wiki/Levenshtein_distance>`_. This distance refers to the minimum number of single-character edits between two words. Some of those changes:
+
+* Change of a character: ``post`` → ``lost``
+* Removal of a character: ``eggs`` → ``ggs``
+* Insertion of a character:  ``edi`` → ``edit``
+* Transposition of two adjacent characters: ``act`` → ``cat``
+
+
+The queries can be constructed as:
+
+.. code-block:: python
+
+     query_body = {
+                    "query": {
+                        "fuzzy": {
+                            field: {
+                                "value": value
+                                "fuzziness": fuzziness,
+                            }
+                        }
+                    }
+                  } 
+
+We can try out looking for a misspelled word and allowing some ``fuzziness``. Writing a fuzzy query with a **misspelled word**, such as ``pinapple`` and setting ``fuzziness`` to zero. Running it, will bring no results:
+
+.. code-block:: python
+
+    python search.py fuzzy "title" "pinapple" 0
+
+
+To correct ``pinapple`` → ``Pineapple`` word, we only need to change one letter. So we can try again to search this word setting the ``fuzziness`` to one and run the search again.
+
+.. code-block:: python
+  
+  python search.py fuzzy "title" "pinapple" 1
+
+As you can see, this search returns results 🍍:
+
+.. code-block:: python
+
+  [
+    'Pineapple "Lasagna" ',
+    'Pineapple Bowl ',
+    'Pineapple Paletas ',
+    'Pineapple "Salsa" ',
+    'Pineapple Sangria ',
+    'Pineapple Tart ',
+    'Pineapple Split ',
+    'Roasted Pineapple with Star Anise Pineapple Sorbet ',
+    'Pineapple-Apricot Salsa ',
+    'Pineapple Papaya Relish '
+  ]
+
+It is your turn, try out more combinations to better understand the fuzzy query.
+
+
+Pause services
+''''''''''''''
+
+After following this tutorial, if you want to give a pause in your service, for the time being, see :doc:`how you can pause the service <../../../platform/howto/pause-from-cli>`. 
+
+What's next?
+''''''''''''
+
+Want to try out OpenSearch with other clients? You can learn how to write search queries with NodeJS client, see :doc:`our tutorial <opensearch-and-nodejs>`.
+
+Resources
+'''''''''
+
+We created an OpenSearch cluster, connected to it, and tried out different types of search queries. Now, you can explore more resources to help you to learn other features of OpenSearch and its Python client.
+
```

Comment:
This is too many links, pick your three favourites and just use those (at least one Aiven link please)